### PR TITLE
calculator list component

### PR DESCRIPTION
**Pull request by jordanlkfung**

### CODE REVIEW

File: src/views/components/Calculator.tsx [hunk 1]
```diff
@@ -2,21 +2,50 @@ import React, { useState } from 'react';
...
             <CalculatorInput onSelect={handleSelect} />
-            {/* <CalculatorItemsView items={items}></CalculatorItemsView> */}
+            <CalculatorItems items={items} onDelete={handleDelete} onUpdate={handleUpdate}></CalculatorItems>
             <CalculatorGraph items={items}></CalculatorGraph>
-            {/* <CalculatorContactButton /> */}
         </Container>
     );
 }
 
+// export function handle(id: number, updatedItem: CalculationDetails) {
```
Comment (NoahCardoza) [inline src/views/components/Calculator.tsx]: If we aren't using this, it's best to keep commented code out of source control. 

Protip: this is one of the things we actually look for when evaluating people at MLH.

File: src/views/components/CalculatorItems.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import {
+    SelectChangeEvent, Autocomplete, TextField,
+    FormControl, InputLabel,
+    Select, MenuItem, Grid
+} from '@mui/material';
+import { modifiers } from "./CalculatorInput"
+// import { handle } from './Calculator';
```
Comment (NoahCardoza) [inline src/views/components/CalculatorItems.tsx]: Same here.

**Review by NoahCardoza — APPROVED** [review]
Just remove the commented code and it LGTM! Great work!

**Comment by Rha02** [comment]
Don't forget to update UI. The "Edit" button should be white, and the length of green background of list items should match that of our form

**Pull request merged**